Projects/Payment/Tests/ServiceTests/PaymentServiceTests.swift

```diff
@@ -60,27 +60,27 @@ final class PaymentServiceTests: XCTestCase {
         assert(respondedPaymentStatusData == paymentStatusData)
     }
 
-    func testFetchPaymentDiscountsDataSuccess() async {
-        let paymentDiscountsData: PaymentDiscountsData = .init(
-            discounts: [],
-            referralsData: .init(
-                code: "code",
-                discountPerMember: .init(amount: "10", currency: "SEK"),
-                discount: .init(amount: "10", currency: "SEK"),
-                referrals: []
-            )
-        )
-
-        let mockService = MockPaymentData.createMockPaymentService(
-            fetchPaymentDiscountsData: { paymentDiscountsData }
-        )
-        self.sut = mockService
-
-        let respondedPaymentDiscountsData = try! await mockService.getPaymentDiscountsData()
-        assert(respondedPaymentDiscountsData == paymentDiscountsData)
-    }
-
-    func testFetchPaymentHistoryDataSuccess() async {
+    //    func testFetchPaymentDiscountsDataSuccess() async throws {
```

Comment:
can we remove this?

Reply:
Just placed it back :) forgot to uncomment it 🙇‍♂️